crime-means-assessment/src/main/resources/schemas/apiCreateMeansAssessmentRequest.json

```diff
@@ -19,9 +19,16 @@
     },
     "userId": {
       "type": "string",
-      "description": "User  ID"
+      "description": "User ID"
+    },
+    "reservationId": {

```

Comment:
This is the same as the RepID, I've created a separate UserSession object in the changes I've made in another branch, so I can sort this out when I merge.